salt/data-pipeline/bigquery-views.sls

```diff
@@ -6,13 +6,13 @@ bigquery-view docker image:
         - unless:
             - test -d /vagrant
 
-re-materialise views daily:
+re-materialise views hourly within working hours:
     cron.present:
         - user: {{ pillar.elife.deploy_user.username }}
         - name: docker run --rm -v /srv/nifi/conf/gcs.json:/root/.config/gcs.json -e GOOGLE_APPLICATION_CREDENTIALS=/root/.config/gcs.json -e DATA_PIPELINE_BQ_PROJECT=elife-data-pipeline elifesciences/data-pipeline-bigquery-views:{{ pillar.data_pipeline.bigquery_views.revision }} ./views-cli.sh --dataset={{ pillar.elife.env }} materialize-views
-        - identifier: materialize-views-daily
-        # at 06:00am UTC every day
-        - hour: "6"
+        - identifier: materialize-views-hourly
```

Comment:
if you change the identifier, you have to add a separate state with `cron.absent` and the old identifier.
Therefore, use a stable name for the new identifier to allow easier future changes, e.g. just `materialize-views`